.github/workflows/rust.yml

```diff
@@ -1,35 +1,40 @@
 name: Rust
 on:
+  # triggers when a PR is posted
   pull_request:
     branches: [develop]
+    paths-ignore:
+      - '**.md'
```

Comment:
This should skip the workflow if someone is just making a markdown change in a PR. The workflow will still occur on merge, unless we want it to be skipped there too.

Reply:
Would a change to a markdown file ever affect the cache?  Seems like we wouldn't need it.

Reply:
I can't imagine how it would impact a build, so I'm not opposed to adding it. If we have it check on push it would simply be out of an extreme abundance of caution.